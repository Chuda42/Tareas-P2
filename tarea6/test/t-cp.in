# prueba de tiempo colaDePrioridad
tiempo_cp 1000000
Fin
