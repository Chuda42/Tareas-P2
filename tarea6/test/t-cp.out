1>#  prueba de tiempo colaDePrioridad.
2>
 prueba 1. 
 prueba 2. 
 prueba 3. 
 prueba 4. 
3>Fin.
